resources/dicts/events/death/beach.json

```diff
@@ -613,87 +646,44 @@
         "m_c": {
             "age": [
                 "adolescent"
             ],
             "status": [
                 "apprentice"
             ],
-            "relationship_status": [],
+            "relationship_status": [ "can_romance" ],
             "dies": true
         },
         "r_c": {
-            "age": [
-                "adolescent"
-            ],
-            "status": []
-        },
-        "history": {
-            "reg_death": "m_c was caught unprepared out in the heat, dying due to dehydration and heatexhustion."
-        },
-        "relationships": [
-            {
-                "values": [],
-                "amount": 5
-            }
-        ]
-    },
-    {
-        "event_id": "bch_death_",
-        "biome": [
-            "beach"
-        ],
-        "camp": [
-            "any"
-        ],
-        "season": [
-            "greenleaf",
-            "newleaf",
-            "leaf-fall"
-        ],
-        "tags": [],
-        "weight": 20,
-        "event_text": "m_c had invited r_c out days earlier, aiming to train and grow closer together. However, when r_c arrived, m_c never showed up; m_c wasn't seen around camp, nor had {PRONOUN/m_c/subject} been to training. It wasn't until r_c stumbles upon {PRONOUN/m_c/poss} body, the former succumbing to dehydration and exhaustion, that {PRONOUN/r_c/subject} understands why.",
-        "m_c": {
             "age": [
                 "adolescent"
             ],
             "status": [
                 "apprentice"
-            ],
-            "relationship_status": [],
-            "dies": true
-        },
-        "r_c": {
-            "age": [
-                "adolescent"
-            ],
-            "status": []
-        },
-        "history": {
-            "reg_death": "m_c was ill-prepared for the heat and succumbed to dehydration."
+            ]
         },
-        "relationships": [
+        "history": [
             {
-                "values": [],
-                "amount": 5
-            }
-        ]
+            "cats": [ "m_c" ],
+            "reg_death": "m_c was ill-prepared for the heat and succumbed to dehydration."
+        }
+    ]
     },
     {
-        "event_id": "bch_death_",
-        "biome": [
-            "beach"
-        ],
+        "event_id": "bch_death_tree_anyleader",
+        "biome": [ "beach" ],
         "camp": [
             "any"
         ],
         "season": [
             "any"
         ],
-        "tags": [],
+        "tags": [
+            "some_lives"
+        ],
         "weight": 20,
         "event_text": "m_c lost a life when {PRONOUN/m_c/subject} fell from the top branches of a tree.",
```

Comment:
I'm almost certain that you'll find this same exact event text in some of the other biome jsons.  One of the perks of our new system is that, instead of having "copies" of an event in multiple biome directories, we can instead stick them in the general json with the proper biome parameters.  

It'll be kinda annoying to find all these duplicate events, but I'd suggest doing a ctrl+shift+f in the death folder whenever you find an event text that doesn't sound particularly biome specific.  If the event is present in more than one biome json, then move it into the general json instead and include the relevant biomes in the biome parameter.  Does this make sense?  Let me know in the discord if I've explained this badly lmao